react/features/base/react/components/native/NavigateSectionList.js

```diff
@@ -274,6 +279,34 @@ export default class NavigateSectionList extends Component<Props> {
         return null;
     }
 
+    _renderListEmptyComponent: () => Object
+
+    /**
+     * Renders a component to display when the list is empty.
+     *
+     * @private
+     * @param {Object} section - The section being rendered.
+     * @returns {React$Node}
+     */
+    _renderListEmptyComponent() {
+        const { onRefresh } = this.props;
+
+        if (typeof onRefresh === 'function') {
+            return (
+                <View style = { styles.pullToRefresh }>
+                    <Text style = { styles.pullToRefreshText }>
+                        Pull to refresh
```

Comment:
Needs to go through translation?

Reply:
You didn't see this

https://media.giphy.com/media/l2JJKs3I69qfaQleE/giphy.gif